throttle.py

```
@@ -0,0 +1,29 @@
+"""
+Utility module for rate limiting requests.
+"""
+import time
+from typing import Optional
+
+
+class RequestThrottler:
+    """Controls request rate to prevent overloading websites."""
+    
+    def __init__(self, requests_per_second: float = 1.0):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)